assets/tests/factories/finishedDetourFactory.ts

```diff
@@ -1,8 +1,23 @@
 import { Factory } from "fishery"
-import { FinishedDetour, RouteSegments } from "../../src/models/detour"
+import {
+  FinishedDetour,
+  RouteSegments,
+  UnfinishedDetour,
+  UnfinishedRouteSegments,
+} from "../../src/models/detour"
 import { shapePointFactory } from "./shapePointFactory"
 import stopFactory from "./stop"
 
+export const unfinishedRouteSegmentsFactory =
```

Comment:
The name of this file is misleading now.

Should I:
a) Separate `unfinishedRouteSegmentsFactory` and `unfinishedDetourFactory` out to a different factory file (thus leaving this file unchanged); OR
b) Keep these as-is, but rename the file to `detourFactory.ts`?

Reply:
I like `b)`!

Reply:
[Will do this as a follow-up](https://app.asana.com/0/1205718271156548/1207845782956705/f)